docs/blog/whats-new-in-phab-3-8.md

```diff
@@ -0,0 +1,265 @@
+configuration and options from the fabfile and you can concentrate on the commands you want to pass to restic. Some
+examples:
+
+```shell
+phab -cmyconfig restic snapshots
+phab -cmyconfig restic -- snapshots --host-name my-config
+phab -cmyconfig restic -- forget --keep-daily=14 --keep-weekly=4 --keep-monthly=6 --group-by host --prune
+```
+
+### Experimental integration with ddev
+
+ddev is a superb tool for managing local development environments and provides similar features as phabalicious. As it
+takes a lot of time to maintain a local development stack (like multibasebox) we thought it might make more sense to
+embrace existing open-source tools.
+
+As ddev is also using docker and docker-compose under the hood it was very unproblematic to integrate ddev with phabalicious.
+Phab will read the ddev configuration from `.ddev/config.yaml` and provide it under the global property `ddev`. So it's
+very easy to provide a phab configuration for your local ddev instance. Instead of copy-pasting the boilerplate into
+every project we store it on a central place and inherit from it in the fabfile:
+
+Store this file in a central place, e.g. in your user-folder or on a remote-server. For demonstration purposes we store
+this in the same folder as the fabfile
+
+the `ddev.yml`-file:
+
+```yaml
+requires: 3.8.0
+
+dockerHosts:
+  ddev:
+    rootFolder: .
+    runLocally: true
+
+
+hosts:
+  ddev:
+    additionalNeeds:
+      - ddev
+    info:
+      description: Local ddev instance
+      publicUrls:
+        - "https://%settings.ddev.name%.ddev.site"
+      category:
+        id: 01-local
+        label: Local development
+    executables:
+      drush: /var/www/html/vendor/bin/drush
+
+    shellProvider: docker-exec
+    composerRootFolder: /var/www/html
+    gitRootFolder: /var/www/html
+    rootFolder: /var/www/html/web
+    backupFolder: /var/www/backups
+    docker:
+      configuration: ddev
+      name: ddev-%settings.ddev.name%-web
+      projectFolder: .
+```
+
+the actual `.fabfile.yml`:
+
+```yaml
+inheritsFrom:
+  - ./ddev.yml
+
+hosts:
+  ddev:
+    # its a drupal installation, so add the drupal-specfic stuff
+    adminPass: admin
+    replaceSettingsFile: false
+    alterSettingsFile: false
+    executables:
+      drush: /var/www/html/vendor/bin/drush
+    needs:
+      - git
+      - docker
+      - drush
+      - files
+```
+
+running `phab list:hosts` will show sth like this for your ddev-project:
+
+```shell
+❯ phab list:hosts -v
+
+Available configurations for your project
+=========================================
+
+Local development
+-----------------
+
+ ‣ ddev
+   Local ddev instance
+   → https://my-ddev-site.ddev.site
+```
+
+You can run any command against the ddev site as usual. For controlling app lifetime please use ddev, e.g. `ddev start`
+or `ddev stop`
+
+
+
+### Smaller enhancements
+
+* Passwords are now obfuscated in phabs output.
```

Comment:
Nitpick, is it `obfuscated` or `redacted` ?

Reply:
I am going with obfuscated for now